code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -552,13 +552,22 @@
 	return H
 
 /mob/living/silicon/pai/MouseDrop(atom/over_object)
-	var/mob/living/carbon/H = over_object
-	if(!istype(H) || !Adjacent(H)) return ..()
-	if(H.a_intent == I_HELP)
-		get_scooped(H)
-		//return
-	else
-		return ..()
+  var/mob/living/carbon/H = over_object
+  if(!istype(H) || !Adjacent(H))  return ..()
+  if(usr == src)
+    switch(alert(H, "[src] wants you to pick them up. Do it?",,"Yes","No"))

```

Comment:
There's no title argument. I left it like that for consistency with other alerts, like the give verb.